lib/trailblazer/operation.rb

```diff
@@ -66,6 +80,16 @@ def run(*params)
 
       [process(*params), valid?].reverse
     end
+    
+    def fetch_collection(*params)
+      setup_collection!(*params)
+      
+      @collection = fetch(*params)
+      @search = perform_search(*params) if self.respond_to?(:perform_search)
```

Comment:
thats ok... I create a blank `perform_search`, and override inside `Pagination` class, like [model!](https://github.com/apotonick/trailblazer/blob/master/lib/trailblazer/operation.rb#L96), [setup_model!](https://github.com/apotonick/trailblazer/blob/master/lib/trailblazer/operation.rb#L100) and [setup_params!](https://github.com/apotonick/trailblazer/blob/master/lib/trailblazer/operation.rb#L103) on [Operation Class](https://github.com/apotonick/trailblazer/blob/master/lib/trailblazer/operation.rb)... seems a good solution?